autoload/lsp/utils.vim

```diff
@@ -167,7 +167,12 @@ function! lsp#utils#get_buffer_path(...) abort
 endfunction
 
 function! lsp#utils#get_buffer_uri(...) abort
-    return lsp#utils#path_to_uri(expand((a:0 > 0 ? '#' . a:1 : '%') . ':p'))
+    let l:name = expand((a:0 > 0 ? '#' . a:1 : '%') . ':p')
+    if empty(l:name)
+        let l:nr = bufnr(a:0 > 0 ? '#' . a:1 : '%')
```

Comment:
`#` means alternate buffer in parameter of `bufnr()` or `bufname()`. So `#{number}` is not valid format here.

```suggestion
        let l:nr = a:0 > 0 ? a:1 : bufnr('%')
```

would be correct